lib/http/client/method.js

```diff
@@ -44,7 +44,7 @@ class Method {
     const { headers, timeout } = option
     return wrapper(
       axios({
-        method: 'post',
+        method: 'put',
```

Comment:
you should improve the tests to catch this  bug in future

Reply:
added a test to check for the method exactly